docs/source/devinstall.md

```diff
@@ -32,10 +32,10 @@ clean                          Make a clean source tree
 dev                            Make a server in jupyter_websocket mode
 docker_clean                   Remove docker images
 docker_clean_enterprise-gateway Remove elyra/enterprise-gateway:dev docker image
-docker_clean_hadoop-spark      Remove elyra/hadoop-spark:2.7.0-2.1.0 docker image
-docker_clean_nb2kg             Remove elyra/nb2kg:dev docker image
+docker_clean_yarn-spark        Remove elyra/yarn-spark:2.1.0 docker image
 docker_image_enterprise-gateway Build elyra/enterprise-gateway:dev docker image
-docker_image_hadoop-spark      Build elyra/hadoop-spark:2.7.1-2.1.0 docker image
```

Comment:
Similar comments here on the usage of underscore and dashes

Reply:
Convention was to use dashes where the artifact name appears since that's the name of the artifact and dashes separating target words.
I figured you'd have an issue with that. I can change to underscores if you want.

Reply:
Convention was to use dashes where the artifact name appears since that's the name of the artifact and dashes separating target words.
I figured you'd have an issue with that. I can change to underscores if you want.

Reply:
@lresende - I found a style guide for make targets which stated that target names should be separated by hyphens ('-'), so I've pushed a corresponding update.